util/pertubation.py

```diff
@@ -0,0 +1,13 @@
+import tensorflow as tf
+import librosa
+import numpy as np
+
+def speed_pertub(samples):
+    samples = samples.flatten().astype('float32')
+    speed_range = np.random.uniform(low=0.8, high=1.4)
+    samples = librosa.effects.time_stretch(samples, rate=speed_range)
+    return samples.reshape([-1, 1])
+
+def pertub(samples):
+    samples = tf.py_func(speed_pertub,[samples], tf.float32)
```

Comment:
This should be `tf.compat.v1.py_func`. Convention in the repo is to import it as `tfv1`. See DeepSpeech.py for example.